app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -599,4 +599,16 @@ public static void playWithKore(final Context context, final Uri videoURL) {
         intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         context.startActivity(intent);
     }
+
+    /**
+     * finish provided Activity and then start MainActivity.
+     *
+     * @param activity the activity to finish
+     * @param context the context to use when creating the Intent
```

Comment:
```suggestion
     * Finish provided <code>Activity</code> and then start <code>MainActivity</code>.
     *
     * @param activity the activity to finish
     * @param context the context to use when creating the <code>Intent</code>
```

Haven't checked this; should wait for CI.